### ISSUE TITLE
Installation problems

### ISSUE THREAD

**Issue by TorstenWuestenberg** [ssciwr/cnsr#12]
Hello Dominik,

when I tried to install the cnsr package under Windows11, the installation froze ... even after an hour of waiting nothing happened. Can you help with this? Maybe I've just made a mistake. You know we use the Anaconda distribution ...

**Comment by dokempf** [comment]
Was the installation triggered by clicking on the app symbol (Kachel) or manually? If the former, yoj are running into the exact same problem I had with the app...

**Comment by TorstenWuestenberg** [comment]
YES. Does this mean that the Anaconda distribution under Windows is not well suited for the cnsr package and that we should always install and run it via a terminal?

**Comment by dokempf** [comment]
By "manual", I do mean either terminal or by selecting it from the package list in the Anaconda UI and install it (like we also did when I visited). Basically, anything but the app symbol. For now, you should advertise these two ways to your users and I would expect both of them to work. Concerning the app, I will try to find help from Anaconda/conda-forge people.
